src/mpi-base.jl

```diff
@@ -526,3 +527,17 @@ function ExScan{T<:MPIDatatype}(object::T, op::Op, comm::Comm)
     sendbuf = T[object]
     ExScan(sendbuf,1,op,comm)
 end
+
+# Conversion between C and Fortran Comm handles:
+
+if sizeof(CComm) == sizeof(Cint)
```

Comment:
Good point.  I guess `cmake` can check whether the functions exist.  (I don't want to use `dlsym_e` because of the upcoming changes in #90.)